src/api/utils.ts

```diff
@@ -0,0 +1,44 @@
+const defaultApiEndpoint = 'http://localhost:4000'
+
+export const get = async (endpoint: string): Promise<any> => {
+  return await fetch(`${defaultApiEndpoint}/${endpoint}`, {
+    method: 'get',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
+}
+
+export const post = async (endpoint: string, body: any): Promise<any> => {
+  return await fetch(`${defaultApiEndpoint}/${endpoint}`, {
+    method: 'post',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(body)
+  })
+}
+
+export const put = async (endpoint: string, body: any): Promise<any> => {
+  return await fetch(`${defaultApiEndpoint}/${endpoint}`, {
+    method: 'put',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(body)
+  })
+}
+
+export const deleteMeth = async (endpoint: string, body: any): Promise<any> => {
```

Comment:
why deleteMeth? What it means?
I understand that delete is reserved word in js, but if you decided to create "api" it would be better to have methods like deleteMovie, addMovie, etc inside it. In you case the same could be achieved with axios library and interceptors to axios config that add default headers and url (ex: https://axios-http.com/docs/config_defaults)